editors/code/src/browser.ts

```diff
@@ -0,0 +1,21 @@
+import * as vscode from 'vscode';
+import { LanguageClient } from "vscode-languageclient/browser";
+
+export function activate(context: vscode.ExtensionContext) {
+    const serverMain = vscode.Uri.joinPath(
+        context.extensionUri,
+        "dist/browserServerMain.js",
+    );
+
+    const worker = new Worker(serverMain.toString(true));
+    worker.onmessage = (message) => {
+        if (message.data !== "OK") {
+            return;
+        }
+
+        const cl = new LanguageClient("sqruff-lsp", "Sqruff LSP", { documentSelector: [{ language: "sql" }] }, worker);
+        cl.start().then(() => { });
+    };
+}
```

Comment:
I am struggling to see how you are testing this in the browser. Did you run `npm run run-in-browser`? and what before?

Reply:
`npm run build:wasm_lsp && npm run compile && npm run run-in-browser`